apps/studio/src/features/editing-experience/components/LayoutSelection.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { Meta, StoryFn } from '@storybook/react'
+import type { LayoutSelectionProps } from './LayoutSelection'
+import LayoutSelection from './LayoutSelection'
+
+export default {
+  title: 'Components/LayoutSelection',
+  component: LayoutSelection,
+  argTypes: {},
+  parameters: {},
+} as Meta
+
+// Template for stories
+const Template: StoryFn<LayoutSelectionProps> = (args) => (
+  <LayoutSelection {...args} />
+)
+
+// Default scenario
+export const Default = Template.bind({})
+Default.args = {
+  pageName: 'Sample Page',
+  pageUrl: '/sample-page',
+  siteId: '1',
+  folderId: '1',
+}
```

Comment:
comment: Might want to use CSFv3 to write stories going forward. (I'll update the rest of the stories in this repo in a future PR)


```suggestion
import { Meta, StoryObj } from '@storybook/react'
import LayoutSelection from './LayoutSelection'

const meta: Meta<typeof LayoutSelection> = {
  title: 'Components/LayoutSelection',
  component: LayoutSelection,
}
export default meta;
type Story = StoryObj<typeof LayoutSelection>

export const Default: Story = {
  args : {
    pageName: 'Sample Page',
    pageUrl: '/sample-page',
    siteId: '1',
    folderId: '1',
    }
  }
```